register/templates/vote.html

```diff
@@ -0,0 +1,83 @@
+{% extends "base.html" %}
+{% load humanize %}
+{% load static %}
+{% block head %}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+{% endblock %}
+{% block body %}
+    <div class="row margin-top">
+        <div class="col-md-6 col-md-offset-3">
+            {% if app %}
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        {# We set the title as lennyface, in out efforts to anonimize the data.. Okay, maybe no #}
+                        <h4 class="text-center">{{ app.lennyface }}</h4>
+                    </div>
+                    <div class="panel-body">
+                        <h4 class="text-center">Personal</h4>
+                        <dl class="dl-horizontal">
+                            {% include 'include/field.html' with desc='Name' value=app.name|add:', '|add:app.lastname %}
```

Comment:
The format here should be either "LastName, Name" or "Name LastName" but not "Name, LastName"